exonum-java-binding/core/pom.xml

```diff
@@ -203,6 +204,7 @@
                 <argument>--color</argument>
                 <argument>always</argument>
               </arguments>
+              <skip>${skipTests}</skip>
```

Comment:
It **must** be built even when we skip tests, but do compile. If you need to temporarily skip it, please add a *separate* property controlling that.